include/seqan3/io/stream/iterator.hpp

```diff
@@ -19,6 +21,7 @@
 #endif // __cpp_lib_ranges
 
 #include <seqan3/core/platform.hpp>
+#include <seqan3/range/views/drop.hpp>
```

Comment:
Where is this used?

Reply:
It was used in your version of write_range. I forgot to remove it again.

Reply:
it is used now